styleguide/javascript/README.md

```diff
@@ -0,0 +1,147 @@
+# JavaScript Styleguide
+
+## Tooling
+
+### Avoid jQuery
+
+Just don't use it. It's better to use standard APIs.
+
+[Should you use or learn jQuery in 2020?](https://flaviocopes.com/jquery/)
+
+### Avoid lodash
+
+You probably don't need it. And if you do, import the functions you need like this:
+
+``` javascript
+import foo from 'lodash/foo'
+```
+
+Don't do this:
+
+``` javascript
+import { foo } from 'lodash'
+import _ from 'lodash'
+```
+
+The latter will make tree-shaking impossible.
+Read more about [lodash and tree-shaking](https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/)
+
+### Use Prettier
+
+Use our [prettier-config](https://github.com/barsoom/prettier-config)
```

Comment:
So this is just something you add to package.json in the project? Any editor tooling needed as well?

Reply:
Yes, it says in the README how you use the config. Then you need to [configure prettier to run](https://prettier.io/docs/en/precommit.html).

There are probably plugins for all editors, but none are needed. Easiest is to just set up a pre-commit-hook

Reply:
In future PRs to the devbook, we can assist users with specific guidance.

Reply:
@aalin I will take the radical step and make those two repositories public, for ease of use.

Reply:
DONE.